### ISSUE TITLE
Student: Personal Information: Child or Youth Content and functional fixes

### ISSUE THREAD

**Issue by astridSABC** [bcgov/SIMS#3353]
**User Story**
As _student user_, I want instructions for child or youth under government care that I can provide SABC the information it needs and I get the funding I need.

**Acceptance Criteria**

- [ ] Revise text "Were you ever considered a child or youth under government care?" **to** "Are you, or were you ever, considered a child or youth under government care?"
- [ ] Add "Prefer not to answer" radio button

<img width="1094" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLzRrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--db071493203a97b3778a91554cbc33f754b06cc7/image.png" alt="image.png" />

- [ ] Revise help info text for "Were you ever considered a child or youth under government care?" from "Ministry of Children and Family Development (MCFD) ...outside of BC" **to**  
 
"(ie. Ministry of Children and Family Development (MCFD) or Indigenous Child and Family Service Agency [ICFSA] care, care through the Ministry of Social Development and Poverty Reduction [SDPR] Child in the Home of Relative Program or government care through a Canadian province or territory outside of B.C.) 
For a variety of reasons, the government may provide care or guardianship for a child. Answer 'YES' to this question if you were ever considered a child or youth under government care (this includes out of care and temporary care statuses as well as continuing custody orders both in B.C. and in Canadian provinces or territories outside of B.C.). If you indicate `YES' to this question, are attending an eligible B.C. institution and you received care or guardianship from the B.C. government for 24 months (730 days) or at the time you turned 19, you may be eligible for the Provincial Tuition Waiver Program for current and former children or youth in care and $3,500 of non-repayable grant assistance through the Learning for Future Grant annually. If you indicate 'YES' to this question, the B.C. Ministry of Children and Family Development may be asked to verify your status as a current or former youth in care and your time in care through the B.C. government for the purpose of assessing your eligibility for the Provincial Tuition Waiver Program and the Learning for Future Grant. "

- [ ] Revise help info text for "Are you, or were you at the time of your 19th birthday....B.C.?" from "This means that the provincial government is currently your legal guardian) ...birthday" **to**  

"(This means that either now or at the time of your 19th birthday the provincial government is/was your legal guardian.) 

For a variety of reasons, the B.C. government may become the legal guardian for a child. If you are currently or were a youth in continuing care/custody of a director of child welfare in B.C. on your 19th birthday, answer 'YES' to this question.  If you indicate `YES' to this question the B.C. Ministry of Children and Family Development may be asked to verify your status as a current or former youth in continuing care or custody of a director of child welfare in B.C."

Refer to [Personal Info Complete.pdf](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2drQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6c70b8556316b3fad4ab55400c9b97573217bf54/Personal%20Info%20Complete.pdf)

**Comment by astridSABC** [comment]
@JerPearson for your review.  Recommend adding bullets to break up the text in the long text in the "Were you ever considered" help text.

**Comment by JerPearson** [comment]
Question to be removed from app.  See ticket #3384

**Comment by JerPearson** [comment]
Waiting on question and tool tip rewrite from policy

**Comment by HRAGANBC** [comment]
Closing as it's being merged into #2826 @JerPearson  FYI
